client/examples/node_api_indexer/07_get_random_basic_outputs.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Calls `GET api/indexer/v1/outputs/basic`.
+//! Run: `cargo run --example node_api_indexer_get_random_basic_outputs --release -- [NODE URL]`.
+
+use iota_client::{node_api::indexer::query_parameters::QueryParameter, Client, Result};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Take the node URL from command line argument or use one from env as default.
+    let node_url = std::env::args().nth(1).unwrap_or_else(|| {
+        // This example uses dotenv, which is not safe for use in production.
+        dotenv::dotenv().ok();
+        std::env::var("NODE_URL").unwrap()
+    });
+
+    // Create a client with that node.
+    let client = Client::builder()
+        // The node needs to have the indexer plugin enabled.
+        .with_node(&node_url)?
+        .finish()?;
+
+    // Get a single page with random output IDs by providing only `QueryParameter::Cursor(_)`.
```

Comment:
What page would that be ?

Reply:
The first one

Reply:
Do we (client) convert empty cursor to first page or is that an indexer thing ?

Reply:
Indexer uses the first page by default, unless you provided a valid cursor for another page